Selenium_Basics/EpamNavigationTests2.cs

```diff
@@ -0,0 +1,109 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenium_Basics;
+
+public class EpamNavigationTests2
+{
+    private IWebDriver driver { get; set; }
+
+    [SetUp]
+    public void SetupBrowser()
+    {
+        var options = new ChromeOptions();
+        options.BinaryLocation = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+        driver = new ChromeDriver(options);
+    }
+    
+    [TearDown]
+    public void TearDown()
+    {
+        driver.Quit();
+    }
+    
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        var epamUrl = "https://www.epam.com/";
+
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//*[text()='Careers']//ancestor::*[@class='top-navigation__item-text']")).Click();
+        driver.FindElement(By.XPath("(//*[name()='use'])[10]")).Click();
+        bool Americas = driver.FindElement(By.XPath("//a[@data-item='0']")).Displayed;
```

Comment:
Please follow naming. Local variables should start with lower-case letter. Also please name variable in order to make it represent true/false logically. Example -> americasElementText, isAmericasArticleDisplayed - you can see thet 1st variable is text, 2nd is bool without doublecheking its values